larpandoracontent/LArVertex/CandidateVertexCreationAlgorithm.cc

```diff
@@ -109,12 +112,37 @@ void CandidateVertexCreationAlgorithm::SelectClusters(ClusterVector &clusterVect
         ClusterVector sortedClusters(pClusterList->begin(), pClusterList->end());
         std::sort(sortedClusters.begin(), sortedClusters.end(), LArClusterHelper::SortByNHits);
 
+        int count1(0);
+        if (m_reducedCandidates)
+        {
+            for (const Cluster *const pCluster : sortedClusters)
+            {
+                float factor1=1;
```

Comment:
Typically initialise using copy construction, and make it clear that this is a floating point literal. So `float factor(1.f)`

Could it be e.g. `selectionCutFactor`, rather than `factor1`?